debian/k8s-cni/Makefile

```diff
@@ -1,7 +1,15 @@
 CNI_RELEASE = 8a936732094c0941e1543ef5d292a1f4fffa1ac5
+PACKAGE = k8s-cni
+VERSION = 0.0.1-0
+DESTINATION = debian/changelog
+DISTRO = xenial
+
+export VERSION PACKAGE DESTINATION DISTRO
+
 
 bin/%:
 	curl -sSL --fail --retry 5 https://storage.googleapis.com/kubernetes-release/network-plugins/cni-$(CNI_RELEASE).tar.gz | tar xz
```

Comment:
please multiple arches here. if you want me to do it, just ping

also, it might be an idea to build the binaries dockerized (reproducable), so we don't depend on someone pushing them to that bucket first.